### ISSUE TITLE
Cannot set property '__init' of undefined

### ISSUE THREAD

**Issue by carlgarton** [artem-sedykh/mini-climate-card#2]

**Comment by stast1** [comment]
У меня та же проблема.
![image](https://user-images.githubusercontent.com/48147949/83728406-04b7c100-a64f-11ea-8fe6-423fa5612bb4.png)

**Comment by cjsmns** [comment]
i'm also getting this red screen.

**Comment by artem-sedykh** [comment]
Sorry, a new release has been released today, thanks for contacting!

**Comment by artem-sedykh** [comment]
If you specify the data for the fan, then there will be no error in the current version  
Today I will release a correction and improvements
```yaml
- type: custom:mini-climate
  entity: climate.dahatsu
  fan_mode:
    source:
      auto: auto
      low: low
	  ....
```

**Comment by artem-sedykh** [comment]
fixed in [v1.0.2](https://github.com/artem-sedykh/mini-climate-card/releases/tag/v1.0.2)
